doc/lua_api.txt

```diff
@@ -2634,6 +2633,19 @@ Some types may inherit styles from parent types.
 * tabheader
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
     * textcolor - color. Default white.
+* box
+    * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+        * Defaults to false in formspec_version version 3 or higher
```

Comment:
Please also document this in `networkprocol.h`.

Reply:
Darn it, why do I have to document what should have already been documented in other PRs?